### PR TITLE
Doc storybook stories

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx [hunk 1]
```diff
@@ -9,59 +9,48 @@ import {
...
+      control: {
+        type: 'boolean'
+      }
+    },
+    label: {
+      description: 'The label asociated to the input value meaning.'
+    },
+    minChar: {
+      description:
+        'The minimun number of characters required for the input value to be valid.'
```
Comment (pdcastano) [inline src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx:32]: '...for the input value to do the search'

File: src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx [hunk 2]
```diff
@@ -9,59 +9,48 @@ import {
...
   title: 'Components/SFAutcompleteLocation',
   component: SFAutocompleteLocation,
+  parameters: { controls: { sort: 'alpha' } },
   args: {
     label: 'Bagel'
   },
   argTypes: {
+    currentLocation: {
+      description:
+        'If true, the component gets the current location coordinates.'
```
Comment (pdcastano) [inline src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx:19]: '...gets the device's current location coordinates.'

File: src/Components/SFButton/SFButton.stories.tsx [hunk 3]
```diff
@@ -7,30 +7,42 @@ import { SFButton } from './SFButton';
...
+        type: {
+          summary: 'string'
+        }
+      },
+      control: { type: 'text' }
     },
-    sfColor: {
-      defaultValue: 'blue'
+    isLoading: {
+      description: 'Tells whether the component finished loading.'
```
Comment (pdcastano) [inline src/Components/SFButton/SFButton.stories.tsx:31]: If true, displays a spinner inside the button.

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx (hunk 2 above)
Comment (Yopo-Nicolas) [reply src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx]: done

File: src/Components/SFButton/SFButton.stories.tsx (hunk 3 above)
Comment (Yopo-Nicolas) [reply src/Components/SFButton/SFButton.stories.tsx]: done

File: src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx (hunk 1 above)
Comment (Yopo-Nicolas) [reply src/Components/SFAutocompleteLocation/SFAutocompleteLocation.stories.tsx]: done

**Review by pdcastano — APPROVED** [review]

**Pull request merged**